### PR TITLE
library hw finished    Dheyazen alseelwe and Danish Khan 

### PR DESCRIPTION
**Pull request by deeworld**

### CODE REVIEW

**Comment by khatchad** [comment]
I don't have a Blackboard submission from your group. Please note the submission instructions and submit the GitHub username of the account you'd like me to grade ASAP:

> ... 
> 1. [Create a **pull request**](https://help.github.com/articles/creating-a-pull-request) on the original repository to turn in the assignment.
> 2. Submit your GitHub **username** on [Blackboard](https://bbhosted.cuny.edu/webapps/assignment/uploadAssignment?content_id=_20246794_1&course_id=_1101235_1&assign_group_id=&mode=cpview).

**Comment by deeworld** [comment]
I resubmitted it, my user name is deeworld. Did u receive it?

**Comment by khatchad** [comment]
Yes.

File: Library/src/lab1/Book.java [hunk 1]
```diff
@@ -22,6 +22,8 @@
      */
     public Book(String bookTitle) {
         // TODO: Implement this method
+        title = bookTitle;
+        borrowed = false;
```
Comment (khatchad) [inline Library/src/lab1/Book.java:26]: This is the default actually.

File: Library/src/lab1/Book.java [hunk 2]
```diff
@@ -22,6 +22,8 @@
      */
     public Book(String bookTitle) {
         // TODO: Implement this method
```
Comment (khatchad) [inline Library/src/lab1/Book.java:24]: You can remove these TODO comments once you have done them.

File: Library/src/lab1/Book.java [hunk 3]
```diff
@@ -52,16 +57,17 @@ public boolean isBorrowed() {
...
     public String getTitle() {
         // TODO: Implement this method
+        return title;
     }
 
     public static void main(String[] arguments) {
         // Small test of the Book class
         Book example = new Book("The Da Vinci Code");
         System.out.println("Title (should be The Da Vinci Code): " + example.getTitle());
-        System.out.println("Borrowed? (should be false): " + example.isBorrowed());
```
Comment (khatchad) [inline Library/src/lab1/Book.java:61]: :-1: Per the instructions:

> You must **not** modify the main method

File: Library/src/lab1/Library.java [hunk 4]
```diff
@@ -6,6 +6,78 @@
  */
 public class Library {
     // TODO: Add the missing implementation to this class
+      String address;
```
Comment (khatchad) [inline Library/src/lab1/Library.java:9]: :-1: Point reduction for invalid source code formatting.

**Comment by khatchad** [comment]
:+1: Good.

**Pull request closed without merging**